modules/cnf-programming-the-sriov-fec-operator-acc100.adoc

```diff
@@ -0,0 +1,350 @@
+// CNF-1498 Validate and Document Intel SRO and SRIOV FEC Operator
+// Module included in the following assemblies:
+//
+// *cnf-optimize-data-performance-n3000.adoc
+
+[id="cnf-programming-the-sriov-operator-acc100_{context}"]
+= Configuring the SR-IOV-FEC Operator for the Intel vRAN Dedicated Accelerator ACC100
+
+Programming the Intel vRAN Dedicated Accelerator ACC100 exposes the Single Root I/O Virtualization (SRIOV) Virtual Function (VF) devices that are then used to accelerate the FEC in the vRAN workload.
```

Comment:
Lowercase "virtual function" because it is not a proper noun.  Apologies for missing this earlier.